.concourse/suse-buildpacks-ci/buildpack-version-bump/tasks/build.sh

```diff
@@ -24,8 +24,9 @@ echo "${REGISTRY_PASS}" | docker login "${REGISTRY_NAME}" --username "${REGISTRY
 # Extract the fissile binary.
 tar xvf s3.fissile-linux/fissile-*.tgz --directory "/usr/local/bin/"
 
-# Pull the stemcell image.
-stemcell_version="$(cat s3.stemcell-version/"${STEMCELL_VERSIONED_FILE##*/}")"
+# Pull the stemcell image referenced in the KUBECF_VALUES
+sle_version="$(cat s3.stemcell-version/"${STEMCELL_VERSIONED_FILE##*/}" | cut -d- -f1)"
```

Comment:
It would be good to have an example of the expected value, so it's easier to reason about.